src/Dialect/ONNX/Rewrite.td

```diff
@@ -329,4 +333,14 @@ def ConstantOpNormalizationPattern6: Pat<
    [(AttributeIsNotNull:$stringsAttr)]
 >;
 
+def ONNXSqueezeOpLowering: Pat<
+   (ONNXSqueezeOp $data, $axes),
+   (ONNXSqueezeV11Op $data, (createArrayAttrFromConstantOp $axes))
+>;
```

Comment:
Please add a condition to check whether `$axes` is produced by ONNXConstantOp, e.g.
```
def ONNXSqueezeOpLowering: Pat<
    (ONNXSqueezeOp $data, $axes),
    (ONNXSqueezeV11Op $data, (createArrayAttrFromConstantOp $axes)),
    [(IsDenseONNXConstant $axes)]
 >;
```

We have a function helper `isDenseONNXConstant`, please write a wrapper function `IsDenseONNXConstant`. Thanks!